### ISSUE TITLE
Cannot implicitly convert type 'System.Collections.ArrayList' to 'System.Collections.Generic.Dictionary<string,object>'

### ISSUE THREAD

**Issue by thristhart** [nomadyow/Ocellus#3]
I get this issue whenever I try to interact with the Frontier API.
Here's my debug log: [debug.log](https://github.com/nomadyow/Ocellus/files/299364/debug.log.txt)

The error is:

```
6/5/2016 10:08 | ERROR: Unable to parse Companion API output Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: Cannot implicitly convert type 'System.Collections.ArrayList' to 'System.Collections.Generic.Dictionary<string,object>'
   at CallSite.Target(Closure , CallSite , Object )
   at Companion.updateProfile(Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues)
```

**Comment by damonbrodie** [comment]
Thanks!  I'm working on this now.

**Comment by damonbrodie** [comment]
Fixed in 0.9
